osu.Game/Screens/Select/Footer.cs

```diff
@@ -124,5 +123,13 @@ public Footer()
 
             updateModeLight();
         }
+        
+        protected override bool InternalContains(Vector2 screenSpacePos) => base.InternalContains(screenSpacePos) || StartButton.Contains(screenSpacePos);
+        
+        protected override bool OnMouseDown(InputState state, MouseDownEventArgs args) => true;

```

Comment:
To disallow clicking through the footer container. Without those you select a beatmap through the footer.